src/datasets/utils/file_utils.py

```diff
@@ -1172,7 +913,7 @@ def _prepare_single_hop_path_and_storage_options(
         client_kwargs = storage_options.pop("client_kwargs", {})
         storage_options["client_kwargs"] = {"trust_env": True, **client_kwargs}  # Enable reading proxy env variables
         if "drive.google.com" in urlpath:
-            response = http_head(urlpath)
+            response = requests.head(urlpath, timeout=10)
```

Comment:
Same comment as above about `requests.head` vs `get_session().head`. Maybe worth doing a pass on the `datasets` codebase in a separate PR

Reply:
they are the only uses of `requests` in `datasets` left :) I'll use `get_session().head` though it's not a big deal imo

Reply:
great to know!